windows.md

```diff
@@ -54,15 +54,60 @@ With those compatibility things out of the way, you're ready to start the system
    ```bash
    choco install slack -y
    ```
-7. Copy the following text and right-click in the blue middle part of the PowerShell window to paste the text. Hit enter.<br><br>
+7. Close PowerShell and open it again as administrator (like in step 2). Copy the following text and right-click in the blue middle part of the PowerShell window to paste the text. Hit enter.<br><br>
    ```bash
    corepack enable
    corepack prepare pnpm@latest --activate
    Set-ExecutionPolicy -Scope Process -ExecutionPolicy Bypass
    pnpm setup
    ```
    This uses Corepack to install `pnpm`, and configures `pnpm`'s global bin directory.<br><br>
-8. Close PowerShell and open it again as administrator (like in step 2). Copy each line in the following text, right-click in the blue middle part of the PowerShell window to paste the text and hit enter.
+   Install `@upleveled/preflight`, a program we will use in the course, to verify that the previous commands were successful: copy the following text, paste it in the terminal and hit return.<br><br>
```

Comment:
seems like we should have empty lines around the paragraphs and code block, right?

Reply:
Added empty lines around the codeblock
[d295f33](https://github.com/upleveled/system-setup/pull/91/commits/d295f336fbe786aae30ba1358e97b15c8c00e0df)